packages/compass-signtool/src/signing-clients/local-signing-client.ts

```diff
@@ -0,0 +1,56 @@
+import path from 'path';
+import { exec } from 'child_process';
+import { promisify } from 'util';
+import { debug } from '../utils';
+import type { SigningClient, SigningClientOptions } from '.';
+
+const execAsync = promisify(exec);
+
+export class LocalSigningClient implements SigningClient {
+  constructor(private options: SigningClientOptions) {}
+
+  private async init() {
+    const remoteScript = `${this.options.rootDir}/garasign.sh`;
+    await execAsync(`mkdir -p ${this.options.rootDir}`);
```

Comment:
Most of these `execAsync` calls could be `fs` calls, e.g. `fs.mkdir()` with `{ recursive: true }`, `fs.chmod()`, `fs.copyFile()`, etc. (and they probably should be unless we really intentionally want the bash extensions to apply here, which I'm pretty sure we don't)